### DISCUSSION TITLE
Error Step 3 - Bad indentation - line 10

### DISCUSSION THREAD

**Discussion by pradorodriguez** [skills/.github#159] [General]
Team,

In Step 3 (https://github.com/skills/hello-github-actions#keyboard-activity-add-actions-steps-to-your-workflow-file), there is a bad indentation in line 10 (- run).
The indentation of the first run should be at the same level as steps, like this:

name: Post Welcome Comment
on:
  pull_request:
    types: [opened]
jobs:
  build:
    name: Post welcome comment
    runs-on: ubuntu-latest    
    steps:
    - run: gh pr comment $PR_URL --body "Welcome to the repository Shinobis!"
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        PR_URL: ${{ github.event.pull_request.html_url }}

**Comment by cmwilson21** [comment]
@pradorodriguez Thanks for opening a discussion to surface this. 👍 

Would you mind opening this up as an issue in the repo you were having trouble with? That way, we can review it and make any adjustments necessary ✨
